### ISSUE TITLE
Doesn't connect via bluetooth

### ISSUE THREAD

**Issue by welowis** [ReimuNotMoe/FiiO-BTR3-USB-Linux-Workaround#1]
Hello,
I want to use Spotify with Fiio BTR3 DAC on Linux(based on debian) system.
Can this changes help to me ?
I could not connect via bluetooth on Linux notebook.
Have a good day..

**Comment by DavidRedstone** [comment]
Fiio BTR3 works for me, with an HP laptop running Fedora 32 (also worked on 30 and 31). 

USB DAC
If you want to use USB DAC mode, which is what this repo is for(!!!) use a USB A to C cable, in addition to following the instructions in this repo. It does not work with a USB C to C cable (and I have never figured out why -- tried multiple USB C-to-C cables, which work with other devices).

Bluetooth
Yes, this works for me. If you want better codec support, see:
https://github.com/EHfive/pulseaudio-modules-bt    
 -- That got APT-X, APT-X HD, and LDAC working for me.

Cheers
Dave

**Comment by welowis** [comment]
> 
> 
> Fiio BTR3 works for me, with an HP laptop running Fedora 32 (also worked on 30 and 31).
> 
> USB DAC
> If you want to use USB DAC mode, which is what this repo is for(!!!) use a USB A to C cable, in addition to following the instructions in this repo. It does not work with a USB C to C cable (and I have never figured out why -- tried multiple USB C-to-C cables, which work with other devices).
> 
> Bluetooth
> Yes, this works for me. If you want better codec support, see:
> https://github.com/EHfive/pulseaudio-modules-bt
> -- That got APT-X, APT-X HD, and LDAC working for me.
> 
> Cheers
> Dave

Hi dear,

Does it work on **Spotify**  via bluetooth ?

Best,

**Comment by DavidRedstone** [comment]
@welowis 
Yes. I use Spotify every day on my phone (Android 10) and my laptop (Ubuntu 20.04).In both cases, Fiio BTR3 works via bluetooth or as a USB-C DAC.
